Packs/Alexa/Integrations/AlexaV2/README.md

```diff
@@ -0,0 +1,106 @@
+Alexa provides website ranking information that can be useful in determining if the domain in question has a strong web presence. V2
+This integration was integrated and tested with Alexa Rank Indicator V2
```

Comment:
Don't forget to add the changes from v1.

Reply:
I copied the changes from the release notes. Is this ok?